test/Validator/Text.php

```diff
@@ -78,8 +78,19 @@
                 (new Text)
                     ->email()
                     ->validate('test.com')
-                    ->errors()
-            )->toBe(['not an email']);
+                    ->errors()[0]
+                    ->__toString()
+            )->toBe('not an email');
+        });
+
+        it('can be translated', function () {
+            expect(
+                (new Text)
+                    ->email()
+                    ->validate('test.com')
+                    ->errors()[0]
+                    ->translate('hello')
```

Comment:
So I was thinking of adding a new method to `Result` that maps over the list of errors and calls translate on each one, with some logic to match tokens. i.e.

```php
$result = $validator->validate($stuff);

$errors = $result->translate(['NOT_EMAIL' => 'hey', 'NOT_ALPHANUMERIC' => 'no']);
```